build/normalize.css

```diff
@@ -0,0 +1,179 @@
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+body {
+  background-color: #FFFFFF;
+  background-repeat: no-repeat;
+  background-image: url("http://www.thatericalper.com/wp-content/uploads/2014/04/Jerryseinfeld.jpg");
+  background-size: 45%;
+  background-attachment: fixed;
+  font-family: 'Roboto Slab', serif;
```

Comment:
background uses double quotes, font uses single. Pick one, and be consistent throughout the file